src/frontend/staff-portal/src/app/components/ticket-page/ticket-page.component.ts

```diff
@@ -281,7 +142,7 @@ export class TicketPageComponent implements OnInit, AfterViewInit {
   getSystemDetectedOcrIssues(ocrViolationTicket?: string): boolean {
     var objOcrViolationTicket = JSON.parse(ocrViolationTicket)
 
-    if (objOcrViolationTicket.fields) {
+    if (objOcrViolationTicket && objOcrViolationTicket.fields) {
```

Comment:
let fields = objOcrViolationTicket?.fields;
if (fields) {      
 keys = Object.keys(fields);
      keys.forEach(key => {
        if (this.getOcrViolationErrors(fields[key]) > 0) {
          return true;
        }
      })
}

Reply:
You can leave this first and you are not on Teams yet